### PR TITLE
Added description modal

### PR DESCRIPTION
**Pull request by ewhanhan**

### CODE REVIEW

File: src/containers/Descriptions/Descriptions.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+class Descriptions extends Component {
+  constructor(props) {
+    super(props);
+    this.myRef = React.createRef();
+  }
+  getRandomInt = (min, max) => {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  };
```
Comment (jhwan7) [inline src/containers/Descriptions/Descriptions.js]: try to remove unused code

File: src/containers/Descriptions/Descriptions.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  scrollToView = () => {
+    console.log(this.myRef);
```
Comment (jhwan7) [inline src/containers/Descriptions/Descriptions.js]: remove console.logs

**Comment by jhwan7** [comment]
image

the button doesnt seem to expand fully across the top

**Comment by jhwan7** [comment]
image
I don't think this quiet matches with the design, only the top of the modal should be displayed on initial page. Also button icon is wrong "+" should be changed to "^"

**Review by jhwan7 — CHANGES_REQUESTED** [review]
try to reflect my comments

File: src/containers/Descriptions/Descriptions.js (hunk 2 above)
Comment (ewhanhan) [reply src/containers/Descriptions/Descriptions.js]: gotcha

**Comment by ewhanhan** [comment]
will be adding pushed footer and change icons shortly

**Review by jhwan7 — APPROVED** [review]

**Pull request merged**